optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -236,7 +236,7 @@ def prepare_inputs_for_generation(
     ) -> Dict:
         return {
             "decoder_input_ids": input_ids,
-            "past_key_values": past,
+            "past_key_values": past_key_values,
```

Comment:
To support `transformers<4.26`, can you add something like:
```suggestion
            "past_key_values": past_key_values or kwargs.get("past", None),
```